src/pages/form/[id].tsx

```diff
@@ -24,13 +35,71 @@ const Form = () => {
       uiSchema: JSON.parse(data.form.uiSchema),
     };
   }, [data]);
-  if (isLoading) return <DefaultLoadingPage />;
-  if (error) return <ErrorPage errorMessage={error.message} />;
+  const user = useSession();
+
+  const wasSubmitted = useMemo(() => {
+    const wasSubmitted = submssionData?.find(
+      (submission) => submission.submittedById === user?.data?.user?.id,
+    );
+    return wasSubmitted;
+  }, [submssionData, user?.data?.user?.id]);
+  const countdownHome = () => {
+    if (countdown <= 0) {
+      router
```

Comment:
just `void router.replace("/")`, it should never throw. it is async but we don't need to wait for it to run